xml/System.Text/Encoding.xml

```diff
@@ -3902,7 +3902,7 @@ In .NET 5.0 and later versions, the code page name `utf-7` is not supported.
  This method returns a list of supported encodings, uniquely distinguished by code page. For a table that lists the supported encodings, see <xref:System.Text.Encoding>.
 
 > [!NOTE]
-> The list of supported encodings returned by the <xref:System.Text.Encoding.GetEncodings%2A> method does not include any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calls to the <xref:System.Text.Encoding.RegisterProvider%2A> method.
+> In .NET Core 3.1 and previous versions, the list of supported encodings that <xref:System.Text.Encoding.GetEncodings%2A> returns does not include any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calling  <xref:System.Text.Encoding.RegisterProvider%2A>. In .NET 5.0 and later versions, the returned list of supported encodings *does* include these additional encodings.
```

Comment:
@gewarren do we need to mention the .NET Framework too?

Reply:
Yes, it's probably better to be explicit about that. How about:
```suggestion
> In .NET 5.0 and later versions, the list of supported encodings that <xref:System.Text.Encoding.GetEncodings%2A> returns includes any additional encodings made available by any <xref:System.Text.EncodingProvider> implementations that were registered by calling <xref:System.Text.Encoding.RegisterProvider%2A>. In previous .NET versions, including .NET Framework and .NET Core, the returned list of supported encodings *does not* include these additional encodings.
```